src/applications/coronavirus-vaccination-expansion/config/military-history/helper.js

```diff
@@ -0,0 +1,32 @@
+const dischargeTypeLabels = {
+  honorable: 'Honorable',
+  general: 'General',
+  other: 'Other Than Honorable',
+  'bad-conduct': 'Bad Conduct',
+  dishonorable: 'Dishonorable',
+  undesirable: 'Undesirable',
+};
```

Comment:
Importing these seems like a good idea rather than redefining them:

https://github.com/department-of-veterans-affairs/vets-website/blob/0677518c2bd7a46e665cd1a54e3c32cb15a63a4e/src/platform/static-data/options-for-select.js#L25-L32